frontend/src/components/board/SidebarItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { MenuItem } from "../../interfaces/canva-interfaces";
+
+interface RenderItemProps {
+  item: MenuItem;
+  activeItem: string | null;
+  color: string;
+  size: number;
+  handleColorChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  handleSizeChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  handleClick: (name: string, action?: () => void) => void;
+}
+
+const RenderItem: React.FC<RenderItemProps> = ({
+  item,
+  activeItem,
+  color,
+  size,
+  handleColorChange,
+  handleSizeChange,
+  handleClick,
+}) => {
+  const id = item.name;
+  return (
+    <div>
+      <button
+        className={`flex items-center text-left text-gray-600 hover:bg-gray-200 p-2 rounded cursor-pointer ${
+          activeItem === item.name ? "bg-gray-200 text-black" : ""
```

Comment:
What is this `activateItem` part for? I don't quite understand.

Reply:
It was used to create a distinctive button that stands out when clicked.